### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by tj** [apex/up#493]
It'd be nice to have something on the site so people can easily stay informed for new releases. Right now you have to follow GH/Slack/Medium pretty closely or you could easily miss it.

**Comment by lukeed** [comment]
Yeah, I'm not big on Medium (putting it nicely). It'd be nice to be informed of minor changes / guides.

**Comment by tj** [comment]
Agreed. I like Medium for more detailed announcements or guides, but for a quick update of changes definitely need something else.

**Comment by kaihendry** [comment]
Wonder if https://github.com/apex/up/releases.atom could be better annotated.

**Comment by tj** [comment]
oh nice I didn't even realize they generated feeds!
